governance.md

```diff
@@ -0,0 +1,53 @@
+# Virtual Kubelet Governance
+
+The following document outlines how the virtual kubelet governance operates. Virtual kubelet relies on two components, the pluggable interface and the providers that "plug-in." 
+
+There are two levels of maintainers for virtual kubelet, core maintainers and project or provider maintainers. 
+
+### Virtual Kubelet Core Maintainers
+
+The Virtual Kubelet Core maintainers are responsible for: 
+
+* Maintaining the mission, vision, values, and scope of the project
+* Refining the governance and charter as needed
+* Making project level decisions
+* Managing the Virtual Kubelet brand
+* Controlling access to Virtual Kubelet assets such as source repositories, hosting, project calendars
+* Handling code of conduct violations
+* Deciding what sub-groups are part of the Virtual Kubelet project
+* Overseeing the resolution and disclosure of security issues
+* Managing financial decisions related to the project
+
+The core maintainers have to match the following criterea: 
+
+* There will be between 3 and 5 people with a diverse representation from multiple companies. No one company shall have a majority vote. 
+* Any project maintainer is eligible to become a core maintainer
+* An core maintainer can step down by emailing our maintainer list (LINK TBA)
+* Core maintainers must remain active, and if they are unresponsive for > 3 months maintainership unless a super-majority of the other core maintainers agree to extend the period to greater than 3 months. 
+* When there is an opening for an core maintainer anyone can nominate candidate as a replacement. 
+    * The nomination period is one month starting the day after the position becomes avaliable. 
+    * The nomination must be via the Virtual Kubelet provider mailing list.
+* When the nominated individual agrees to be a canidate, the project maintainers and the current core maintainers may vote. 
+    * The voting period will be a minimum of 7 days and will close when a majority of project maintainers have voted. 
+    * Each maintainer will get 1 vote, yes or no, and depending on the majority the candidate will either become a core maintainer or not. Maintainers will vote through a election form (Office forms or Google forms).
```

Comment:
Just for fair voting, what do you think about https://civs.cs.cornell.edu/?

Reply:
I'm OK with that.